datahub-web-react/src/app/context/useInitialRedirect.ts

```diff
@@ -19,8 +19,8 @@ export function useInitialRedirect(state, localState, setState, setLocalState) {
                 ...state,
                 loadedInitialPath: true,
             });
-            if (localState.selectedPath) {
-                history.push({
+            if (localState.selectedPath && !localState.selectedPath.includes(PageRoutes.EMBED)) {
```

Comment:
amazing - forgot to bring this to OSS after making this change in saas - thanks!

Reply:
that makes sense